src/Files.Uwp/Strings/en-US/Resources.resw

```diff
@@ -2789,4 +2789,10 @@ We use App Center to track which settings are being used, find bugs, and fix cra
   <data name="ShowAlternateStreams" xml:space="preserve">
     <value>Show Alternate Data Streams</value>
   </data>
+  <data name="ReviewFiles" xml:space="preserve">
+    <value>Review Files</value>
+  </data>
+  <data name="ReviewFilesContent" xml:space="preserve">
+    <value>Would you like to review Files?</value>
```

Comment:
I prefer the current wording as the yes/no action fits with the actual question. Just because someone enjoys Files, they might not be interested in reviewing it.